ui/admin/app/components/form/host-catalog/gcp/index.hbs

```diff
@@ -240,10 +240,36 @@
     {{/if}}
   </Hds::Form::Textarea::Field>
 
+  <Hds::Form::Textarea::Field
+    name='worker_filter'
+    @value={{@model.worker_filter}}
+    @isOptional={{true}}
+    @isInvalid={{@model.errors.worker_filter}}
+    disabled={{form.disabled}}
+    as |F|
+  >
+    <F.Label>{{t 'form.worker_filter.label'}}</F.Label>
+    <F.HelperText>
+      {{t 'resources.host-catalog.form.worker_filter.help'}}
+      <Hds::Link::Inline @href={{doc-url 'worker.create-tags'}}>
+        {{t 'actions.learn-more'}}
+      </Hds::Link::Inline>
+    </F.HelperText>
+    {{#if @model.errors.worker_filter}}
+      <F.Error as |E|>
+        {{#each @model.errors.worker_filter as |error|}}
+          <E.Message>
+            {{error.message}}
+          </E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::Textarea::Field>
+
```

Comment:
If we are adding a worker filter here we might want to use the new worker filter pattern that includes the worker filter generator. [Example here](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/components/form/storage-bucket/aws/index.hbs#L458-L491).
We might want to loop design in on this to though. We have different approaches for worker filters depending on if they are option or not.

Reply:
thank you for linking that, checking with the design team and shall update based on the outcome

Reply:
checked with Design, since AWS host catalog uses the old style, we decided to keep this the same as that for now, but plan to update the host catalog worker filter to the new style. thread [here](https://hashicorp.slack.com/archives/C07E8C15JGL/p1736372414200009?thread_ts=1736287134.480429&cid=C07E8C15JGL)

Reply:
Thanks for the explanation 😃 🙌